guide/canvas/README.md

```diff
@@ -0,0 +1,334 @@
+
+Now, you need to load the image you want to use into canvas. In order to have more sufficient coverage I will first show how to load a basic image from a local directory. We will use this as our background in our welcomer image. In order for this to work, please download [this image](/assets/img/wallpaper.jpg) and store it in the directory where this code is ran, with the name `wallpaper.jpg`.
+
+```js
+const Discord = require('discord.js');
+const Canvas = require('canvas');
+
+client.on('guildMemberAdd', async member => {
+	const channel = member.guild.channels.find(ch => ch.name === 'member-log');
+	if (!channel) return;
+
+	const canvas = Canvas.createCanvas(700, 200);
+	const ctx = canvas.getContext('2d');
+
+	// Since the image takes time to load, we should await it
+	const background = await Canvas.loadImage('./wallpaper.jpg');
+	// This uses the canvas dimensions to stretch the image onto the entire canvas
+	ctx.drawImage(background, 0, 0, canvas.width, canvas.height);
+	// Use helpful Attachment class structure to process the file for you
+	const attachment = new Discord.Attachment(canvas.toBuffer(), 'image.png');
+
+	channel.send(`Welcome to the server, ${member}`, attachment);
+});
+```
+
+If you have that code working it should present you with the following image:
+![Image](/assets/img/8CQvVRV.png)
+
+If you get an error, such as `Error: error while reading from input stream`, then the provided path to the file was incorrect.
+
+Next, let's place a border around the image, just for the sake of it- which is mainly demonstration.
+
+```js
+const Discord = require('discord.js');
+const Canvas = require('canvas');
+
+client.on('guildMemberAdd', async member => {
+	const channel = member.guild.channels.find(ch => ch.name === 'member-log');
+	if (!channel) return;
+
+	const canvas = Canvas.createCanvas(700, 200);
+	const ctx = canvas.getContext('2d');
+
+	const background = await Canvas.loadImage('./wallpaper.jpg');
+	ctx.drawImage(background, 0, 0, canvas.width, canvas.height);
+
+	// Select the colour of the stroke
+	ctx.strokeStyle = '#FF0000';
+	// Draw a rectangle with the dimensions of the entire canvas
+	ctx.strokeRect(0, 0, canvas.width, canvas.height);
+
+	const attachment = new Discord.Attachment(canvas.toBuffer(), 'image.png');
+
+	channel.send(`Welcome to the server, ${member}`, attachment);
+});
+```
+
+If followed successfully, the image should now look like this: ![Image](/assets/img/2vsIPEP.png) 
+
+Quite boring and plain, is it not? Fear not, for you have a bit more to do until we reach completion. Since designing is hard, let's just place a basic square shaped avatar for now on the left side of the image. In interest of coverage, you will also make it a circle afterwards.
```

Comment:
```diff
- Fear not, for you have ...
+ EVERYTHING IS OKAY NOW. WHY? BECAUSE I AM HERE.
```
![WATASHI GA KITA](https://cdn.discordapp.com/emojis/434797270802759680.png?v=1)